Prog/Affordability_sf_condo.sas

```diff
@@ -0,0 +1,321 @@
+		   if PITI_Repeat <= (89300*0.5 / 12*.28) then AMI50_repeat_afford=1; else AMI50_repeat_afford=0; 
+	end;
+	if saleyear =2004 then do;
+           if PITI_First <= (85400*0.8 / 12*.28) then AMI80_first_afford=1; else AMI80_first_afford=0; 
+		   if PITI_First <= (85400*0.5 / 12*.28) then AMI50_first_afford=1; else AMI50_first_afford=0; 
+		   if PITI_Repeat <= (85400*0.8 / 12*.28) then AMI80_repeat_afford=1; else AMI80_repeat_afford=0; 
+		   if PITI_Repeat <= (85400*0.5 / 12*.28) then AMI50_repeat_afford=1; else AMI50_repeat_afford=0; 
+	end;
+	if saleyear =2003 then do;
+           if PITI_First <= (84800*0.8 / 12*.28) then AMI80_first_afford=1; else AMI80_first_afford=0; 
+		   if PITI_First <= (84800*0.5 / 12*.28) then AMI50_first_afford=1; else AMI50_first_afford=0; 
+		   if PITI_Repeat <= (84800*0.8 / 12*.28) then AMI80_repeat_afford=1; else AMI80_repeat_afford=0; 
+		   if PITI_Repeat <= (84800*0.5 / 12*.28) then AMI50_repeat_afford=1; else AMI50_repeat_afford=0; 
+	end;
+	if saleyear =2002 then do;
+           if PITI_First <= (91500*0.8 / 12*.28) then AMI80_first_afford=1; else AMI80_first_afford=0; 
+		   if PITI_First <= (91500*0.5 / 12*.28) then AMI50_first_afford=1; else AMI50_first_afford=0; 
+		   if PITI_Repeat <= (91500*0.8 / 12*.28) then AMI80_repeat_afford=1; else AMI80_repeat_afford=0; 
+		   if PITI_Repeat <= (91500*0.5 / 12*.28) then AMI50_repeat_afford=1; else AMI50_repeat_afford=0; 
+	end;
+	if saleyear =2001 then do;
+           if PITI_First <= (85600*0.8 / 12*.28) then AMI80_first_afford=1; else AMI80_first_afford=0;
+		   if PITI_First <= (85600*0.5 / 12*.28) then AMI50_first_afford=1; else AMI50_first_afford=0;
+		   if PITI_Repeat <= (85600*0.8 / 12*.28) then AMI80_repeat_afford=1; else AMI80_repeat_afford=0;
+		   if PITI_Repeat <= (85600*0.5 / 12*.28) then AMI50_repeat_afford=1; else AMI50_repeat_afford=0;
+	end;
+	if saleyear =2000 then do;
+           if PITI_First <= (82800*0.8 / 12*.28) then AMI80_first_afford=1; else AMI80_first_afford=0;
+		   if PITI_First <= (82800*0.5 / 12*.28) then AMI50_first_afford=1; else AMI50_first_afford=0;
+		   if PITI_Repeat <= (82800*0.8 / 12*.28) then AMI80_repeat_afford=1; else AMI80_repeat_afford=0;
+		   if PITI_Repeat <= (82800*0.5 / 12*.28) then AMI50_repeat_afford=1; else AMI50_repeat_afford=0;
+	end;
+
+end;
+
+run;
+proc contents data=create_flags;
+run;
+
+proc print data= create_flags (obs=25);
+var saleprice saleyear PITI_FIRST PITI_repeat AMI80_first_afford AMI50_first_afford AMI80_repeat_afford AMI50_repeat_afford;
+run;
+proc freq data=create_flags; 
+tables AMI80_first_afford AMI50_first_afford AMI80_repeat_afford AMI50_repeat_afford; 
+run;
+*proc summary at city, ward levels - so you could get % of sales in Ward 7 affordable to 
+median white family vs. median black family.;
+
+/*Proc Summary: Affordability for Owners by 80AMI and 50AMI*/
+
+proc summary data=create_flags;
```

Comment:
@maggiesu0725  - don't need city variable (they are all DC).

Reply:
@maggiesu0725 please add a proc sort before the proc summaries and change the class statement to a by statement. then you won't have all the extra groupings.